dev/com.ibm.ws.security.common/src/com/ibm/ws/security/common/structures/CacheValue.java

```diff
@@ -10,16 +10,26 @@
  *******************************************************************************/
 package com.ibm.ws.security.common.structures;
 
-public class CacheEntry {
+import java.util.Objects;
+
+public class CacheValue {
 
     private Object value;
     private long createdAt = 0L;
+    private long clockSkew = 0L;
 
-    public CacheEntry(Object value) {
+    public CacheValue(Object value) {
         this.value = value;
         this.createdAt = System.currentTimeMillis();
     }
 
+    public CacheValue(Object value, long clockSkew) {
+        this(value);
+        if (clockSkew > 0) {
+            this.clockSkew = clockSkew;
+        }
+    }
+
```

Comment:
I suggest something like:

```
    public CacheValue(Object value) {
        this(value, 0);
    }

    public CacheValue(Object value, long clockSkew) {
        this.value = value;
        this.createdAt = System.currentTimeMillis();
        if (clockSkew > 0) {
            this.createdAt -= clockSkew;
        }
    }
```
This avoids having to store clockSkew and calculate below.